### ISSUE TITLE
MotionSensor doesn't notify both high and low interrupts

### ISSUE THREAD

**Issue by user2684** [mysensors/NodeManager#152]
```
int sensorPIR_Id = nodeManager.registerSensor(SENSOR_MOTION,3);
  SensorMotion* sensorMotion = (SensorMotion*)nodeManager.getSensor(sensorPIR_Id);
  sensorMotion->setMode(CHANGE);
  sensorMotion->setInitial(LOW);
//  sensorMotion->setDebounce(50);
//  sensorMotion->setTriggerTime(1000);
```

**Comment by user2684** [comment]
Working in my tests, waiting for additional info

**Comment by user2684** [comment]
Fixed with #151
